### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ps0305**
#### Instructions

Please read and follow the instructions before creating and submitting a pull request:

- Create an issue explaining the feature. It could save you some effort in case we don't consider it should be included in axios.
- If you're fixing a bug, try to commit the failing test/s and the code fixing it in different commits.
- Ensure you're following our [contributing guide](https://github.com/axios/axios/blob/master/CONTRIBUTING.md).

**⚠️👆 Delete the instructions before submitting the pull request 👆⚠️**

Describe your pull request here.

### CODE REVIEW

**Comment by mnewelski** [comment]
The irony of a PR to the README file that doesn't follow the PR easy to read PR instructions.

File: README.md [hunk 1]
```diff
@@ -171,7 +176,7 @@ axios('/user/12345');
 
 ### Request method aliases
 
-For convenience aliases have been provided for all supported request methods.
+For convenience, aliases have been provided for all supported request methods.
```
Comment (chinesedfan) [inline README.md]: here

File: README.md [hunk 2]
```diff
@@ -36,6 +36,11 @@ Using npm:
 ```bash
 $ npm install axios
 ```
+Using yarn:
+
+```bash
+$ yarn add axios
+```
```
Comment (chinesedfan) [inline README.md]: here

**Comment by chinesedfan** [comment]
Closed due to stale.

**Pull request closed without merging**